src/actions/receive-profile.js

```diff
@@ -1049,7 +1049,7 @@ export function getProfilesFromRawUrl(
     }
 
     return {
-      profile: getProfile(getState()),
+      profile: getProfileOrNull(getState()),
```

Comment:
I had to change this because of the `from-addon` since it receives the profile asynchronously. Otherwise tests were failing, although we don't actually use that data when it's `from-addon`. I'm not super happy with this change :/

Reply:
Oof, that is a bit unfortunate. Could you add a code comment about when we can expect the profile to exist, and when we shouldn't?

Reply:
Added the comments.